ucte/ucte-network/src/main/java/com/powsybl/ucte/network/io/UcteWriter.java

```diff
@@ -111,23 +103,33 @@ private void writeLineBlock(UcteRecordWriter writer) throws IOException {
         LOGGER.trace("Writing line block");
         writer.writeString("##L", 0, 3);
         writer.newLine();
-        for (UcteLine l : network.getLines()) {
-            writeElementId(l.getId(), writer);
-            writer.writeInteger(l.getStatus().getCode(), 20);
-            writer.writeFloat(l.getResistance(), 22, 28);
-            writer.writeFloat(l.getReactance(), 29, 35);
-            writer.writeFloat((float) (l.getSusceptance() / Math.pow(10, -6)), 36, 44);
-            writer.writeInteger(l.getCurrentLimit(), 45, 51);
-            writer.writeString(l.getElementName(), 52, 64);
+
+        network.getLines().stream().sorted((line1, line2) -> line1.getId().toString().compareTo(line2.getId().toString()))
```

Comment:
Same here, wouldn't it be better to make UcteElementId implement Comparable interface and use something like:
`
network.getLines().stream().sorted(Comparator.comparing(UcteLine::getId))...
`